DependencyInjection/Configuration.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
```

Comment:
File headers like this should be removed. (At least I started to remove them from all files, because it does not contain any relevant information IMHO). Only author docblocks should be preserved on classes.